apps/backend/src/app/api/v1/auth/passkey/register/verification-code-handler.tsx

```diff
@@ -0,0 +1,178 @@
+import { prismaClient } from "@/prisma-client";
+import { createVerificationCodeHandler } from "@/route-handlers/verification-code-handler";
+import { VerificationCodeType } from "@prisma/client";
+import { verifyRegistrationResponse } from "@simplewebauthn/server";
+import { KnownErrors } from "@stackframe/stack-shared";
+import { yupMixed, yupNumber, yupObject, yupString } from "@stackframe/stack-shared/dist/schema-fields";
+import { StackAssertionError } from "@stackframe/stack-shared/dist/utils/errors";
+import { RegistrationResponseJSON } from "@stackframe/stack-shared/dist/utils/passkey";
+import {decodeClientDataJSON} from "@simplewebauthn/server/helpers";
+
+export const registerVerificationCodeHandler = createVerificationCodeHandler({
+  metadata: {
+    post: {
+      summary: "Set a new passkey",
+      description: "Sign in with a passkey",
+      tags: ["Passkey"],
+      hidden: true,
+    }
+  },
+  type: VerificationCodeType.PASSKEY_REGISTRATION_CHALLENGE,
+  requestBody: yupObject({
+    credential: yupMixed<RegistrationResponseJSON>().required(),
+    code: yupString().required(),
+  }),
+  data: yupObject({
+    challenge: yupString().required(),
+    userHandle: yupString().required(),
+  }),
+  method: yupObject({
+  }),
+  response: yupObject({
+    statusCode: yupNumber().oneOf([200]).required(),
+    bodyType: yupString().oneOf(["json"]).required(),
+    body: yupObject({
+      user_handle: yupString().required(),
+    }),
+  }),
+  async send() {},
```

Comment:
This should never get called, right? We should throw an assertion error

```suggestion
  async send() {
    throw new StackAssertionError("send() called on a Passkey registration verification code handler")
  },
```